src/solver/src/simulation_builder.C

```diff
@@ -33,11 +33,12 @@
 namespace GRINS
 {
   SimulationBuilder::SimulationBuilder()
-    : _mesh_builder( new MeshBuilder ),
-      _vis_factory( new VisualizationFactory ),
-      _qoi_factory( new QoIFactory ),
-      _postprocessing_factory( new PostprocessingFactory )
-  {}
+  {
+    _mesh_builder = std::make_shared<MeshBuilder>();
+    _vis_factory = std::make_shared<VisualizationFactory>();
+    _qoi_factory = std::make_shared<QoIFactory>();
+    _postprocessing_factory = std::make_shared<PostprocessingFactory>();
```

Comment:
These can't stay in the initializer list?

Reply:
For some reason, I had it in my (apparently increasingly dysfunctional...) head that you couldn't pass those to constructors. Thanks for pointing it out.